### ISSUE TITLE
TypeError: Incorrect value of args option

### ISSUE THREAD

**Issue by dasantonym** [pm2-hive/pm2-server-monit#10]
just installed and i am getting this exception on ubuntu 14.04.1 with node v0.10.31 and pm2 v0.14.3:

```
TypeError: Incorrect value of args option
    at exports.spawn (child_process.js:715:11)
    at Object.execute (/home/web/.pm2/node_modules/pm2-server-monit/lib/process.js:25:17)
    at Netstat.module.exports.Netstat.start (/home/web/.pm2/node_modules/pm2-server-monit/lib/network.js:60:27)
    at Object. (/home/web/.pm2/node_modules/pm2-server-monit/lib/netstat.js:11:9)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function._load (/usr/lib/node_modules/pm2/node_modules/pmx/lib/transaction.js:62:21)
    at Module.require (module.js:364:17)
```

is there any necessary configuration or requirements i am missing?

**Comment by jonkafton** [comment]
+1

**Comment by alavit-d** [comment]
Fixed - Missing argument in child_process.spawn() function failed in older node.js version.

**Comment by Unitech** [comment]
published 0.7.0, you can update it from keymetrics or do a `pm2 install pm2-server-monit`
